src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
Part of the reason why arj didn't make it as a popular web compression method is the same as LZH where it wasn't parse-able without downloading the entire archive. PK *(zip)* is and has been to my knowledge since the olden days. Similar to how ID3 v1.0/1.1 data blocks for mp3's is at the tail end of the binaries vs v2.x which is at the forefront.

I even tried it on a clean Apache with zero mods and it comes with no `Content-Type` header on a response. Not sure if I have my IIS available from that time period of MSDN to see if anything was defaulted but I doubt it.